backend/src/src-codescoop/src/main/java/org/eclipse/sw360/codescoop/CodescoopHandler.java

```diff
@@ -48,27 +48,41 @@
     public CodescoopHandler() {
         Properties props = CommonUtils.loadProperties(CodescoopHandler.class, PROPERTY_FILE);
         String host = props.getProperty(PROPERTY_HOST, null);
-        String client = props.getProperty(PROPERTY_CLIENT, null);
+        String user = props.getProperty(PROPERTY_CLIENT, null);
         String credentials = props.getProperty(PROPERTY_CREDENTIALS, null);
         String proxy = props.getProperty(PROPERTY_PROXY, null);
 
         log.info("Codescoop client construct : host=" + host + ", client=" + client
                 + ", credentials=" + credentials + ", proxy=" + proxy);
 
+        if (isNotBlank(host) && isNotBlank(user) && isNotBlank(credentials)) {
+            while (client == null) {
```

Comment:
Would it be good to have some limit on retries? Like five times or something? Since I have no account and not even the necessary jar for building / trying, I cannot judge if it makes sense to try over and over again and what would be expected by sysadmins if the remote service is not reachable (which might probably be one of the reasons why the client creation fails).